open_academy/wizards/open_academy_wizard.py

```diff
@@ -0,0 +1,17 @@
+from odoo import models, fields, api
+
+
+class OpenAcademyWizard(models.TransientModel):
+    _name = "open.academy.wizard"
+    _description = "Quick registration of attendees to sessions"
+
+    def _default_sessions(self):
+        return self.env["open.academy.session"].browse(self._context.get("active_ids"))
+
+    session_wizard_ids = fields.Many2many('open.academy.session', default=_default_sessions)
+    attendee_wizard_ids = fields.Many2many("res.partner")
```

Comment:
Don't add `wizard` to field's name.

```suggestion
    session_ids = fields.Many2many('open.academy.session', default=_default_sessions)
    attendee_ids = fields.Many2many("res.partner")
```